unnamed_toaster/unnamed_toaster_hw_interface/scripts/ros_table_interface.py

```diff
@@ -63,6 +65,7 @@ def run(self):
 
             self.turret.update()
             self.turret.publish()
+            self.match_logic.run()
```

Comment:
rather than running match logic here, just add a publisher that publishes a std_msgs/String containing the robot_mode, then subscribe to that from the match logic node.

Reply:
Oh, gotcha, that makes a lot of sense wish id thought of that.